homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -376,9 +378,10 @@ def __init__(self, device, logical) -> None:
         """Initialize the device."""
         self._device = device
         self._icon = None
-        self._state = None
+        self._state: str | None = None
         self._logical_address = logical
         self.entity_id = "%s.%d" % (DOMAIN, self._logical_address)
+        self._attr_should_poll = False
```

Comment:
There are other places you can use `_attr` attributes instead of properties; since there's more to do here, I would suggest limiting this PR to just its stated intention (typing).

Reply:
Well the intention of the pr are mypy AND attr. I believe I have converted the variables that make sense to convert, which ones do you think sbout?

Reply:
Let me jus5 walk through again to be sure

Reply:
or I will just limit it as you suggest.

Reply:
Removed in this PR